testing/src/main/java/com/linecorp/decaton/testing/TestUtils.java

```diff
@@ -61,7 +58,7 @@ private static int sequence() {
         return sequence.getAndIncrement();
     }
 
-    private static Properties defaultProducerProps(String bootstrapServers) {
+    public static Properties defaultProducerProps(String bootstrapServers) {
```

Comment:
Is this visibility change still necessary?

Reply:
Oh no, I forgot to revert the change in https://github.com/line/decaton/pull/157/commits/1dfb7a17cba70eb88dffcecfa0745bee66c3f86c. Let me make it `private` again

Reply:
https://github.com/line/decaton/pull/157/commits/bf3d42c6c2b76074d0f759f45d5c16beb0c10a71